src/dagmc/dagmcmetadata.hpp

```diff
@@ -351,11 +360,13 @@ class dagmcMetaData {
   std::map<std::string, std::string> keyword_synonyms;
 
   // Some constant keyword values
-  const std::string graveyard_str{"Graveyard"};
-  const std::string vacuum_str{"Vacuum"};
-  const std::string reflecting_str{"Reflecting"};
-  const std::string white_str{"White"};
-  const std::string periodic_str{"Periodic"};
+  const std::string graveyard_str_{"Graveyard"};
+  const std::string vacuum_str_{"Vacuum"};
+  const std::string vacuum_mat_str_{"mat:Vacuum"};
+  const std::string graveyard_mat_str_{"mat:Graveyard"};
+  const std::string reflecting_str_{"Reflecting"};
+  const std::string white_str_{"White"};
+  const std::string periodic_str_{"Periodic"};
```

Comment:
We frequently - but not always - convert these to lower case.  Is there a reason to not just define them as lower case?

Reply:
I think they are only converted to lower case when comparing with a group name read from a h5 geometry.
My guess is the default if not lower case, but allowing some flexibility for the users.

I don't have preferences there, it is really up to you and Pat :)

@pshriwise as you are the initiator of the branch do you have a take on this ?